obspy/io/gse2/src/GSE_UTI/gse_functions.c

```diff
@@ -207,7 +207,7 @@ int decomp_6b_buffer (int n_of_samples, int32_t *dta, char * (* reader)(char *,
   int i, ibuf=-1, k, inn, jsign=0, joflow=0;
   int32_t itemp;
   char cbuf[83]=" ";
-
+  setbuf(stdout, NULL);
```

Comment:
unfortunatly that didn't do much - I still had multiple env's showing the message